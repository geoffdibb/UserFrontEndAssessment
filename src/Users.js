import React, { Component } from 'react';
import { Table } from 'reactstrap';



class Recipe extends Component {

    render() {

        return (

            <div style={{ display: 'flex', justifyContent: 'center', backgroundColor: "white" }}>


                <Table >

                    <tbody><tr></tr>
                        <tr>
                            <td>{this.props.username}
                            </td>
                        </tr>


                    </tbody>

                </Table>

            </div>

        )
    }
}

export default Recipe;
