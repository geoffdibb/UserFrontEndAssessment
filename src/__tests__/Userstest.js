import React from 'react';
import ReactDOM from 'react-dom';
import Users from '../Users';
import renderer from 'react-test-renderer';


it('snapshot Users', () => {
  const tree = renderer.create(<Users/>).toJSON()
  expect(tree).toMatchSnapshot();
})