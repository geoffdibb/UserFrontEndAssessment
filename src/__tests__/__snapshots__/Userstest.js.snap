// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Users 1`] = `
<div
  style={
    Object {
      "backgroundColor": "white",
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <table
    className="table"
  >
    <tbody>
      <tr />
      <tr>
        <td />
      </tr>
    </tbody>
  </table>
</div>
`;
